charts/tornjak/templates/server-statefulset.tpl

```diff
@@ -127,6 +146,15 @@ spec:
         - name: spire-server-socket
           mountPath: {{ .Values.tornjak.config.backend.socketDir }}
         # livenessProbe:
+        startupProbe:
+          httpGet:
+            scheme: HTTP
+            port: 3000  
+          failureThreshold: 6
+          initialDelaySeconds: 60
+          periodSeconds: 30
+          successThreshold: 1
+          timeoutSeconds: 10
```

Comment:
this is awesome! should upstream to Tornjak quickstart to help with this issue: https://github.com/spiffe/tornjak/issues/102